docs: add release notes for 20250212 clustered release (#5831)
influxdata/docs-v2: content/influxdb3/clustered/reference/release-notes/clustered.md

---

## 20250212-1570743 {date="2025-02-12"}

### Quickstart

```yaml
spec:
  package:
    image: us-docker.pkg.dev/influxdb2-artifacts/clustered/influxdb:20250212-1570743
```

### Bug Fixes

This release fixes a bug in the 20241217-1494922 release where the default 
Prometheus CPU limit was set to an integer instead of a string.

### Changes

#### Deployment

> - The Prometheus `retention` period has been exposed, allowing users 
>   to set a custom retention period for Prometheus metrics.

Reviewer: ```suggestion
- Expose the Prometheus `retention` period to let users set a custom
  retention period for Prometheus metrics.
```

#### Database Engine

> - Datafusion upgrades

Reviewer: ```suggestion
- Upgrade DataFusion
```

> - Added the ability to restore a cluster's data from a Data Snapshot.

Reviewer: ```suggestion
- Add the ability to restore a cluster from a Catalog snapshot.
```
